src/services/TaskRepository.ts

```diff
@@ -1,83 +1,86 @@
+                    [sortBy , sortOrder]
+                ]
+            })
         }
-        
-        var [results] = await instances.dbs.ToDoAPI.sequelize.query(
-            queryString
-        )
-        return results;
+
+        return results
     }
 
-    async fetchTask(id:number) {
-        const [results] = await instances.dbs.ToDoAPI.sequelize.query(
-            'SELECT * FROM Tasks WHERE id = ?',
-            {
-                replacements: [id],
-                type: QueryTypes.SELECT
+    async fetchTask(fetchID:number) {
+        const query = Task.TaskModel()
+
+        const results = query.findAll({
+            where: {
+                id: fetchID
             }
-        )
-        return results;
+        })
+
+        return results
     }
 
     async createTask(desc:string, due:Date, completed: boolean){
-        const newTask:ITask = {
-            id: 0,
+
+        const query = Task.TaskModel()
+
+        const newTask = await query.create({
+            id: uuidv4(),
             taskDescription:desc, 
-            createdAt: new Date(),
             dueDate:due, 
             completed:completed
-        }
-        const created = newTask.createdAt.toISOString().split('T')[0]
-
-        const [results] = await instances.dbs.ToDoAPI.sequelize.query(
-            'INSERT INTO Tasks (taskDescription, createdAt, dueDate, completed) '+
-            `VALUES (?, ?, ?, ?)`,
-            {
-                replacements:[desc, created, due, completed],
-                type: QueryTypes.INSERT
-            }
-        )
-        return results;
+        })
+        return newTask
     }
 
-    async updateTask(id:number, desc:string, due:Date, completed: boolean) {
-        const [results] = await instances.dbs.ToDoAPI.sequelize.query(
-            'UPDATE Tasks SET ' +
-            `taskDescription = ?, ` +
-            `dueDate = ?, ` +
-            `completed = ? ` +
-            `WHERE id = ?`,
-            {
-                replacements: [desc, due, completed, id],
-                type: QueryTypes.UPDATE
+    async updateTask(fetchID:number, desc:string, due:Date, completed: boolean) {
+        const query = Task.TaskModel()
+
+        const updatedTask = await query.update({
+            taskDescription: desc,
+            dueDate: due,
+            completed: completed
+        },
+        {
+            where: {
+                id: fetchID
             }
-        )
-        return results
+        })
+
+        return updatedTask
     }
 
-    async deleteTask(id:number) { 
-        const [results] = await instances.dbs.ToDoAPI.sequelize.query(
-            'DELETE FROM Tasks WHERE id = ?',
-            {
-                replacements: [id],
+    async deleteTask(fetchID:number) { 
+        const query = Task.TaskModel()
+
+        const results = query.destroy({
+            where: {
+                id: fetchID
             }
-        )
-        return results;
+        })
+
+        return results
```

Comment:
Commented about this in Liam's PR too, so destroy will only return the number of rows that had been deleted from the table. I just want you to look at what you want to return in the delete endpoint to be more helpful than returning the number of rows that are no longer with us